R/input-select.R

```diff
@@ -4,7 +4,7 @@
 #' from a list of values.
 #'
 #' By default, `selectInput()` and `selectizeInput()` use the JavaScript library
-#' \pkg{selectize.js} (<https://github.com/selectize/selectize.js>) instead of
+#' \pkg{selectize.js} (<https://selectize.dev/docs/usage>) instead of
```

Comment:
Let's point them at homepage for this one.

```suggestion
#' \pkg{selectize.js} (<https://selectize.dev/) instead of
```